cdc/entry/schema_storage.go

```diff
@@ -267,10 +277,39 @@ func WrapTableInfo(schemaID int64, schemaName string, version uint64, info *timo
 			}
 		}
 	}
-
+	ti.findHandleIndex()
 	return ti
 }
 
+func (ti *TableInfo) findHandleIndex() {
+	if ti.HandleIndexID == -1 {
+		// pk is handle
+		return
+	}
+	handleIndexOffset := -1
+	for i, idx := range ti.Indices {
+		if !ti.IsIndexUnique(idx) {
+			continue
+		}
+		if idx.Primary {
+			handleIndexOffset = i
+			break
+		}
+		if handleIndexOffset < 0 {
+			handleIndexOffset = i
+		} else {
+			if len(ti.Indices[handleIndexOffset].Columns) > len(ti.Indices[i].Columns) ||
+				(len(ti.Indices[handleIndexOffset].Columns) == len(ti.Indices[i].Columns) &&
+					ti.Indices[handleIndexOffset].ID > ti.Indices[i].ID) {
+				handleIndexOffset = i
+			}
+		}
+	}
+	if handleIndexOffset >= 0 {
+		ti.HandleIndexID = ti.Indices[handleIndexOffset].ID
+	}
```

Comment:
Is it better to add fatal log here, if handle index is not found, we can record more information here.

Reply:
I check it here:
https://github.com/pingcap/ticdc/pull/787/files#diff-8cae4fdc1c5f0a4e944025f65917425fR437

Reply:
But the check in mounter can't log enough debug information